projection/src/main/scala/akka/projection/r2dbc/internal/R2dbcOffsetStore.scala

```diff
@@ -685,11 +702,33 @@ private[projection] class R2dbcOffsetStore(
   }
 
   def readManagementState(): Future[Option[ManagementState]] = {
-    Future.successful(None) // FIXME not implemented yet
+    def createStatement(connection: Connection) =
+      connection
+        .createStatement(readManagementStateSql)
+        .bind("$1", projectionId.name)
+        .bind("$2", projectionId.key)
+
+    r2dbcExecutor
+      .selectOne("read management state")(
+        conn => createStatement(conn),
+        row => ManagementState(row.get("paused", classOf[java.lang.Boolean])))
   }
 
   def savePaused(paused: Boolean): Future[Done] = {
-    Future.successful(Done) // FIXME not implemented yet
+    r2dbcExecutor
+      .updateOne("update management state") { conn =>
+        conn
+          .createStatement(updateManagementStateSql)
+          .bind("$1", projectionId.name)
+          .bind("$2", projectionId.key)
+          .bind("$3", paused)
+          .bind("$4", Instant.now(clock).toEpochMilli)
```

Comment:
I tried a few options and the only thing that seems to work is: 

* "$1" with `bind(0, ...)` 
* "$1" with `bind("$1", ...)` 

Using a name like `$projection_name` doesn't work. 

I still don't understand why we prefer to use `bind(0, ...)` or `bind("$1", ...)`, instead of constants with clear names.

Reply:
ok, I only tried `bind("$1", ...)` and assumed it would work for any string.

> I still don't understand why we prefer to use bind(0, ...), instead of constants with clear names.

I don't mind adding constants for the indexes in the bind calls.

For the sql construction I don't think we should use those constants to create the "$1" in the sql string. Instead I still think the search-replace in #135 would be nice.

Reply:
we should not use `.bind("$1"`
that has no value over `.bind(0` and only adds runtime overhead

Reply:
I will replace with `bind(int, ...)`

> For the sql construction I don't think we should use those constants to create the "$1"

My initial idea was to use an constant `Int`, but use it to generate the right $ param and then refer to the constant as usage sight. 

After all the discussion here, it's not clear how you want to implement #135. It seems to me that we will still need to count the order of params to know if we are using the write index. Anyway, maybe better to discuss this further in #135.